admin-ui.php

```diff
@@ -98,29 +99,42 @@ static function admin_page_footer() {
 
 	function admin_page() {
 		// Handle delete request. Will default back to "tokens" later
-		if ( isset( $_REQUEST['action'] ) && 'delete' == $_REQUEST['action'] ) {
-			if ( !isset( $_REQUEST['nonce'] ) || !wp_verify_nonce( $_REQUEST['nonce'], 'keyring-delete-' . $_REQUEST['service'] . '-' . $_REQUEST['token'] ) ) {
+		if ( isset( $_REQUEST['action'] ) && 'delete' === $_REQUEST['action'] ) {
+			if ( ! isset( $_REQUEST['nonce'] ) || ! wp_verify_nonce( $_REQUEST['nonce'], 'keyring-delete-' . $_REQUEST['service'] . '-' . $_REQUEST['token'] ) ) {
 				Keyring::error( __( 'Invalid/missing delete nonce.', 'keyring' ) );
 				exit;
 			}
 
-			if ( $this->keyring->get_token_store()->delete( array( 'id' => (int) $_REQUEST['token'], 'type' => 'access' ) ) )
+			if ( $this->keyring->get_token_store()->delete(
+				array(
+					'id'   => (int) $_REQUEST['token'],
+					'type' => 'access',
+				)
+			) ) {
 				Keyring::message( __( 'That connection has been deleted.', 'keyring' ) );
-			else
+			} else {
 				Keyring::message( __( 'Could not delete that connection!', 'keyring' ) );
+			}
 		}
 
 		// Handle test request. Will default back to "tokens" later
-		if ( isset( $_REQUEST['action'] ) && 'test' == $_REQUEST['action'] ) {
-			if ( !isset( $_REQUEST['nonce'] ) || !wp_verify_nonce( $_REQUEST['nonce'], 'keyring-test-' . $_REQUEST['service'] . '-' . $_REQUEST['token'] ) ) {
+		if ( isset( $_REQUEST['action'] ) && 'test' === $_REQUEST['action'] ) {
```

Comment:
This does change the logic, but I can't think of a case when $_REQUEST['action'] == 'test' is TRUE but  $_REQUEST['action'] === 'test' is FALSE, so I think it's ok.